engines/grim/grim.cpp

```diff
@@ -1117,6 +1117,13 @@ void GrimEngine::mainLoop() {
 			g_imuseState = -1;
 		}
 
+		#if defined(__EMSCRIPTEN__)
```

Comment:
The preprocessor directives must have zero indentation, e.g. start from the column 1 per our code formatting giudelines.